test/staging/Intl402/Temporal/old/non-iso-calendars.js

```diff
@@ -16,70 +36,280 @@ var testChineseData = new Date("2001-02-01T00:00Z").toLocaleString("en-US-u-ca-c
 });
 var hasOutdatedChineseIcuData = !testChineseData.endsWith("2001");
 
+const calendars = [
```

Comment:
Can you use `Object.keys`  on `expected` instead? This runs the risk of people adding expected data but forgetting to update this list as well.

Reply:
Good point, I also added another commit to do this elsewhere in the file.